src/Okta.Sdk.UnitTests/UsersClientShould.cs

```diff
@@ -124,5 +125,464 @@ public async Task RemoveApplicationTargetFromApplicationAdministratorRoleForUser
 
             mockRequestExecutor.ReceivedHref.Should().StartWith("/api/v1/users/foo/roles/bar/targets/catalog/apps/baz");
         }
+
+        [Fact]
+        public async Task ListUserClients()
+        {
+            var rawResponse = @"[
+                                    {
+                                        ""client_id"": ""0oabskvc6442nkvQO0h7"",
+                                        ""client_name"": ""My App"",
+                                        ""client_uri"": null,
+                                        ""logo_uri"": null,
+                                        ""_links"": {
+                                            ""grants"": {
+                                                ""href"": ""https://${yourOktaDomain}/api/v1/users/00u5t60iloOHN9pBi0h7/clients/0oabskvc6442nkvQO0h7/grants""
+                                            },
+                                            ""tokens"": {
+                                                ""href"": ""https://${yourOktaDomain}/api/v1/users/00u5t60iloOHN9pBi0h7/clients/0oabskvc6442nkvQO0h7/tokens""
+                                            }
+                                        }
+                                    }
+                                ]";
+            var mockRequestExecutor = new MockedStringRequestExecutor(rawResponse);
+            var client = new TestableOktaClient(mockRequestExecutor);
+            await client.Users.ListUserClients("foo").ToListAsync();
```

Comment:
Let's check that the response is parsed properly and we get the object we are expecting.